llvm/include/llvm/CodeGen/MachinePassRegistry.def

```diff
@@ -41,6 +41,7 @@ FUNCTION_PASS("consthoist", ConstantHoistingPass, ())
 FUNCTION_PASS("ee-instrument", EntryExitInstrumenterPass, (false))
 FUNCTION_PASS("expand-large-div-rem", ExpandLargeDivRemPass, ())
 FUNCTION_PASS("expand-large-fp-convert", ExpandLargeFpConvertPass, ())
+FUNCTION_PASS("expand-memcmp", ExpandMemCmpPass, (TM))
```

Comment:
Have we settled on a naming scheme for passes yet? I think this would be `expand-mem-cmp` as mem and cmp abbreviate two separate words and that seems to be the style elsewhere.